packages/lit/README.md

```diff
@@ -1,26 +1,63 @@
 # Lit
```

Comment:
I would get rid of the #Lit H1 (since the name is also in the logo, but do add an alt to the MD), and just go with Logo+tagline so it's not duplicated:
![image](https://user-images.githubusercontent.com/1354932/115575345-52ddaf00-a277-11eb-95c4-39b2091e4608.png)

Reply:
done